test/plugins/builds.test.js

```diff
@@ -746,6 +747,161 @@ describe('build plugin test', () => {
                     });
                 });
             });
+
+            describe('join', () => {
```

Comment:
why is this test suite nested inside `buildToken` ?

Reply:
The `PUT /builds/id` has two scopes, `user token` and `build token`. User can only update build status to `ABORTED`. Build can do all the stuff.